### DISCUSSION TITLE
Orchard CMS not loading in localhost anymore

### DISCUSSION THREAD

**Discussion by rrdgcoks** [OrchardCMS/OrchardCore#11471] [General]
Hello! I'm fairly new to Orchard Core. Running in Windows 10 with Chrome by default.

I have a CMS site running in VS 2022 locally. I started up the project yesterday and it won't load in [localhost](https://localhost:44300/) anymore. It just says "Loading..." and is looping without actually loading the site. The only changes I made previously had to do with attempting to publish, but it was all done in the :Publish page. No code changes. 

There are no errors shown. The output for the build says 156 succeeded and 0 failed. 

Does anyone have any ideas on how to proceed? Debug isn't doing anything for me because the site isn't loading. I tried loading the site in Edge but it's not working there either.

Thank you!

**Comment by rrdgcoks** [comment]
I think I know what caused the issue, but not sure why. We moved the Orchard CMS folder to a shared server, instead of having it on a local computer. In doing so, it is failing to open in Visual Studio. It seems you have to install Orchard CMS in a particular location and not move it? All I know is the first location is working in VS 2022 and the second location is not. The database is network connected so I don't think that has anything to do with this. 

If you wish, you can close this as resolved, or help me figure out why I can't move Orchard files once it's built in Visual Studio. Thanks

**Reply by Skrypt** [reply]
Using a network share to build OC is not a great idea because of performance degradation. It must retrieve a fairly massive amount of data through the network before getting it to execute. Also, there are a lot of permissions check that happens for each file which can cause the Visual Studio experience pretty slow.

**Reply by rrdgcoks** [reply]
okay thanks for the information
